installation-wizard-page/src/App.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import { message } from "antd";
+import axios from "axios";
+import "./App.css";
+import "antd/dist/antd.css";
+import Button from "antd/es/button";
+
+function App() {
+  const [instanceId, setInstanceId] = useState("");
+  const [loading, setLoading] = useState(false);
+  const checkInstance = async () => {
+    setLoading(true);
+    axios
+      .post("/v1/AWSInstanceCheck", { instance_id: instanceId })
+      .then(function(response) {
+        // handle success
+        window.location.href = "/";
+      })
+      .catch(function(error) {
+        // handle error
+        message.error(
+          error.response.data.message
+            ? error.response.data.message
+            : error.message
+        );
+      })
+      .finally(() => {
+        setLoading(false);
+      });
+  };
```

Comment:
```suggestion
const checkInstance = async () => {
     setLoading(true);
     try {
      const response = await axios.post("/v1/AWSInstanceCheck", { instance_id: instanceId });
      // handle success
      window.location.href = "/";
     } catch (error) {
         // handle error
         message.error(error.response.data.message ? error.response.data.message : error.message);
     } finally {
     () =>setLoading(false);
     }
   };
```

IMHO, if lambda is `async` I'd preferred to use async/await in whole code. It looks more compact.
What do you think?